Cannot build aobench with ISPC

I tried to build the `aobench` ISPC code, but it failed. The issue seems to be when linking:
```
/usr/bin/ld: objs/ao_ispc_sse2.o: relocation R_X86_64_32S against `.rodata.cst16' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: objs/ao_ispc_sse4.o: relocation R_X86_64_32S against `.data' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: objs/ao_ispc_avx.o: relocation R_X86_64_32S against `.data' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: objs/ao_ispc_avx2.o: relocation R_X86_64_32S against `.rodata.cst32' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: objs/ao_ispc_avx512knl.o: relocation R_X86_64_32S against `.rodata' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: objs/ao_ispc_avx512skx.o: relocation R_X86_64_32S against `.rodata' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: objs/ao_ispc.o: relocation R_X86_64_PC32 against symbol `abort@@GLIBC_2.2.5' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: final link failed: bad value
clang-6.0: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [common.mk:82: ao] Error 1
```

Searching on the internet revealed that the issue might be with my Linux distro (ArchLinux) [enabling PIC for executables by default](https://stackoverflow.com/questions/46827433/g-compile-error-rodata-can-not-be-used-when-making-a-shared-object-recomp).

Adding the `-no-pie` linker flag does help:
```
LIBS=-lm $(TASK_LIB) -lstdc++ -no-pie
```
(I've added it here since there doesn't seem to be a `LDFLAGS` variable).

--- a/examples/aobench/volta/common.mk
+++ b/examples/aobench/volta/common.mk
@@ -8,7 +8,7 @@ CXXFLAGS+=-Iobjs/ -O3 -march=native
 CC=clang
 CCFLAGS+=-Iobjs/ -O3 -march=native
 
-LIBS=-lm $(TASK_LIB) -lstdc++
+LIBS=$(TASK_LIB) -lstdc++ -no-pie
 ISPC=ispc
 ISPC_FLAGS+=-O3
 ISPC_HEADER=objs/$(ISPC_SRC:.ispc=_ispc.h)

